ports/migrations/0001_initial.py

```diff
@@ -0,0 +1,79 @@
+# Generated by Django 2.1.7 on 2019-04-11 13:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BuildHistory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('builder_name', models.CharField(db_index=True, max_length=50)),
+                ('build_id', models.IntegerField()),
+                ('status', models.CharField(max_length=50)),
+                ('port_name', models.CharField(db_index=True, max_length=50)),
+                ('time_start', models.CharField(max_length=50)),
+                ('time_elapsed', models.CharField(max_length=50)),
```

Comment:
These two fields need to represent a real timestamp rather than a string, so that you can potentially sort, do arithmetics, do queries about events that happened on a particular day, ...